frontend/src/metabase/api/database.ts

```diff
@@ -1,10 +1,16 @@
-import type { DatabaseId } from "metabase-types/api";
+import type { DatabaseId, DatabaseResult } from "metabase-types/api";
 
 import { Api } from "./api";
 import { FIELD_VALUES_TAG } from "./tags";
 
 export const databaseApi = Api.injectEndpoints({
   endpoints: builder => ({
+    listDatabases: builder.query<DatabaseResult, void>({
+      query: () => ({
+        method: "GET",
+        url: "/api/database",
+      }),
```

Comment:
Did you consider using `transformResponse` here to normalize response payload?

This way we wouldn't have to alternate the types (`Database | DatabaseEntity`) and we could get rid of `getNormalizedApiDatabases` selector.

Reply:
I didn't want to normalize the response. "Normalize" here means replacing nested entities with their ids, and this is something we want to avoid now